src/handlebars/about.handlebars

```diff
@@ -71,13 +71,12 @@
           </li>
           <li>
             <strong>Can I bundle AdoptOpenJDK binaries in my application and distribute them?</strong><br/>
-              * Hotspot based binaries, Yes. The GPLv2+CE license allows you do this.<br/>
-              * Eclipse OpenJ9 based binaries, Yes. The licenses allow for this.
+              Yes. The GPLv2+CE license allows you do this.
           </li>
           <li>
             <strong>I heard some folks wondering if the lack of TCK/JCK putting AdoptOpenJDK under legal risk.
               Does distributing the AdoptOpenJDK binary (whether part of my app or stand alone) expose me to legal risk?</strong><br/>
-            No. AdoptOpenJDK has the same source code + minor patches (under the same license) as OpenJDK.
+            No. AdoptOpenJDK has the same source code + minor patches (under the same GPLv2+CE) as OpenJDK.
```

Comment:
Suggestion: Either `under GPLv2+CE` or `under the same GPLv2+CE license`

Reply:
fixed